applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -16,6 +16,14 @@
 
 #define LE_AUDIO_SDU_SIZE_OCTETS(bitrate) (bitrate / (1000000 / CONFIG_AUDIO_FRAME_DURATION_US) / 8)
 
+#if (CONFIG_SCAN_MODE_ACTIVE)
+#define BT_LE_SCAN_TYPE    BT_LE_SCAN_TYPE_ACTIVE
```

Comment:
Since these defines are set here, we don't recommend using BT_LE_* as these are defined elsewhere.

Reply:
OK, I'll use a project specific prefix then.